orca/topology/alerts/extractor.py

```diff
@@ -88,9 +89,13 @@ def _mapping(self):
         return self.__mapping
 
     def map(self, name, labels):
-        mapping = self._mapping.get(name)
+        mapping = self._mapping['plain'].get(name)
         if not mapping:
-            raise exceptions.MappingNotFound(key=name)
+            try: 
+                key = [regex for regex in self._mapping['regex'].keys() if re.compile(regex).match(name)][0]
+                mapping = self._mapping['regex'].get(key)
+            except:
+                raise exceptions.MappingNotFound(key=name)
```

Comment:
The `map` methods gets complex.

Let's move the logic responsible for finding the mapping for a given alert name into a separate method, for instance `_get_mapping(name)`.